### ISSUE TITLE
Update user load to use finalized list of patron groups

### ISSUE THREAD

**Issue by dlrueda** [sul-dlss/FOLIO-Project-Stanford#109]
When patron groups and “user group” custom field drop down choices are finalized,  add them into the user load code.

Depends on #37

Check with Laszlo for mapping. (Not sure if the new patron groups will be based on registry affiliation, or if they are SUL-created users so no user load updates needed.  Maybe a mixture of both.)

**Comment by shelleydoljack** [comment]
Duplicate of https://github.com/sul-dlss/folio_api_client/issues/117
